setup.py

```diff
@@ -41,7 +41,7 @@
         'cfgv>=1.0.0',
         'identify>=1.0.0',
         'nodeenv>=0.11.1',
-        'pyyaml',
+        'pyyaml>=4.2b4',
```

Comment:
pyyaml 4.x is very much in flux, and pre-commit works fine with the 3.x version

Fortunately, there's supposed to be a `3.13` release that comes out today -- but isn't out yet https://github.com/yaml/pyyaml/issues/193#issuecomment-402333150